### PR TITLE
New changes for openedx vagrant release

### PR DESCRIPTION
**Pull request by hkim823**

### CODE REVIEW

**Comment by hkim823** [comment]
Also moving aspen Vagrantfile into "base" vagrant folder

**Comment by feanil** [comment]
:+1: to the new-relic change, the open-edx vagrant file should not have a release name as part of its path.

**Comment by hkim823** [comment]
Maybe this should be part of this PR as well.  I'll work on this branch for more changes.

File: playbooks/vagrant-openedx.yml [hunk 1]
```diff
@@ -45,4 +45,3 @@
     - certs
     - role: analytics-api
```
Comment (hkim823) [inline playbooks/vagrant-openedx.yml]: Should this even be here?

**Pull request merged**